README.md

```diff
@@ -497,26 +501,30 @@ See [libFuzzer Tutorial](https://github.com/google/fuzzing/blob/master/tutorial/
 ## Docker Instructions
 
 If you have [Docker](https://www.docker.com/) installed, you can run this
-in your terminal, when the Dockerfile is in your working directory:
+in your terminal, when the Dockerfile is inside the .devconatiner directory:
+
+```bash
+cd .devcontainer/
+```
 
 ```bash
 docker build --tag=my_project:latest .
 docker run -it my_project:latest
 ```
```

Comment:
Instead of cd you can specify the docker file name.
```bash
docker build -f ./.devcontainer/Dockerfile --tag=my_project:latest .
docker run -it my_project:latest
```

Reply:
yhep I will change that!